### PR TITLE
For Story Id 190461 :  Adding sample code to respond to Throttling (429) events.

### PR DESCRIPTION
**Pull request by ritikaKaushal**
For Story Id =190461
   Adding sample code to respond to Throttling (429) events.
And deleting existing samples

### CODE REVIEW

File: cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/APIThrottlingSample.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
+package com.ibm.cnxdevs.cloud.apimanagement.sample;
```
Comment (ManishKataria) [inline cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/APIThrottlingSample.java]: Include the legal header here, you can copy from existing files, adjusting the year to 2016.

File: cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/APIThrottlingSample.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	/*
+	 * Method responsible for making api requests
+	 * Throttling events are handled using exponential backoff algorithm for illustration.
+	 */
+	public void apiRequests(String username, String password) {
+		try {
+			   int retryCount = 0;
+			   boolean retry = false;
+
+				prepareHttpClient(username, password);
```
Comment (ManishKataria) [inline cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/APIThrottlingSample.java]: Can we correct the formatting ? Or is it just git which shows odd formatting all over the class.

File: cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/APIThrottlingSample.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+		}
+	}
+
+	
+	/*
+	 * Returns the next wait interval, in milliseconds
+	 */
+	public static long getWaitTime(int retryCount) {
+
+	    long waitTime = ((long) Math.pow(retryCount, 2) * 1000L);
```
Comment (ManishKataria) [inline cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/APIThrottlingSample.java]: Some Comments on this section please.

File: cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/util/Prop.java [hunk 4]
```diff
@@ -7,5 +7,6 @@
 	public static String appkey = "enter your appkey here";
```
Comment (vfrancis) [inline cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/util/Prop.java]: We should probably put these properties into a .properties file and read that in.

File: cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/APIThrottlingSample.java (hunk 2 above)
Comment (vfrancis) [inline cloud.apimanagement.sample/src/main/java/com/ibm/cnxdevs/cloud/apimanagement/sample/APIThrottlingSample.java]: Agree, we should go through the formatting. I suspect its due to tabs vs spaces and how that renders.

**Pull request merged**